### DISCUSSION TITLE
Add quick selection tool

### DISCUSSION THREAD

**Discussion by unoffending** [opentoonz/opentoonz#6488] [Ideas]
### Is your feature request related to a problem? Please describe

Often, the outline is not completely black, because of this, the shadow you add makes this outline darker, by analogy, when you add highlights, the outlines become lighter. In addition, it is not always convenient to use FX to ensure that shadows do not go beyond the drawing (mattle in FX)

### Describe the solution you'd like

It would be great to add a quick selection tool, such as in Krita or Photoshop (magic wand). This would help make shadows and highlights faster and more conveniently.
I wish the quick selection would still work when you go to another layer, it would help avoid having to pick a darker shade for each color while still using transparent black.

**Comment by oppotaku** [comment]
the fact that this feature hasn't been implemented is the only reason why i haven't fully switched to opentoonz
